src/edu/harvard/CS50.java

```diff
@@ -256,9 +258,13 @@ public static long getLong(String prompt)
      */
     public static String getString(String prompt)
     {
+
+        // instantiate scanner once
+        if (s == null)
+            s = new Scanner(System.in);
```

Comment:
We don't typically use braces when the body is a single line, but sure, if that's what you want. First, are we gonna keep or instantiate upon declaration?